test/cypress/integration/card/test-payment-notification.test.cy.js

```diff
@@ -0,0 +1,157 @@
+const cardPaymentStubs = require('../../utils/card-payment-stubs')
+const { adminUsersGetService } = require('../../utils/stub-builders/service-stubs')
+const {
+  connectorMultipleSubsequentChargeDetails,
+  connectorValidPatchConfirmedChargeDetails
+} = require('../../utils/stub-builders/charge-stubs')
+const { cardIdValidCardDetails } = require('../../utils/stub-builders/card-id-stubs')
+
+const tokenId = 'be88a908-3b99-4254-9807-c855d53f6b2b'
+const chargeId = 'ub8de8r5mh4pb49rgm1ismaqfv'
+const validPayment = {
+  cardNumber: '4444333322221111',
+  expiryMonth: '01',
+  expiryYear: '30',
+  name: 'S. McDuck',
+  securityCode: '012',
+  addressLine1: 'McDuck Manor',
+  city: 'Duckburg',
+  postcode: 'SW1A 1AA',
+  email: 's.mcduck@example.com'
+}
+const createPaymentChargeStubsEnglish = (isLive) => {
+  return cardPaymentStubs.buildCreatePaymentChargeStubs(tokenId, chargeId, 'en', 42, {}, {}, {
+    gatewayAccountType: isLive ? 'live' : 'test'
+  })
+}
+const checkCardDetailsStubs = (isLive) => {
+  return cardPaymentStubs.checkCardDetailsStubs(chargeId, 42, {
+    gatewayAccountType: isLive ? 'live' : 'test'
+  })
+}
+const confirmPaymentDetailsStubs = (isLive) => {
+  const gatewayAccountType = isLive ? 'live' : 'test'
+  return [
+    adminUsersGetService(),
+    connectorMultipleSubsequentChargeDetails([
+      {
+        chargeId,
+        status: 'AUTHORISATION READY',
+        state: { finished: false, status: 'started' },
+        gatewayAccountType
+      },
+      {
+        chargeId,
+        paymentDetails: validPayment,
+        status: 'AUTHORISATION READY',
+        state: { finished: false, status: 'submitted' },
+        gatewayAccountType
+      },
+      {
+        chargeId,
+        paymentDetails: validPayment,
+        status: 'AUTHORISATION SUCCESS',
+        state: { finished: false, status: 'submitted' },
+        gatewayAccountType
+      },
+      {
+        chargeId,
+        paymentDetails: validPayment,
+        status: 'AUTHORISATION SUCCESS',
+        state: { finished: false, status: 'submitted' },
+        gatewayAccountType
+      }
+    ]),
+    cardIdValidCardDetails(),
+    connectorValidPatchConfirmedChargeDetails(chargeId)
+  ]
+}
+
+
+
+describe('Card payment page', () => {
+  describe('A test payment', () => {
+    it('should show phase and notification banners', () => {
+      cy.task('setupStubs', createPaymentChargeStubsEnglish(false))
+      cy.visit(`/secure/${tokenId}`)
+      cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+      shouldCheckPhaseBanner()
+      shouldCheckNotificationBanner()
+      cy.task('clearStubs')
```

Comment:
I think it would be easier to follow if there was a line break each time the stubs are cleared, but that may be a personal thing so I wouldn't hold it back for that reason